ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/converters/ListViewHttpConverter.java

```diff
@@ -89,7 +89,7 @@ public void write(List<ViewSpecification> views, MediaType contentType,
 		rdfModelConverter.checkLangForRelativeUrl(rdfFormat);
 		Model model = ModelFactory.createDefaultModel();
 		views.stream().map(viewSpecificationConverter::modelFromView).forEach(model::add);
-
+		outputMessage.getHeaders().setContentType(MediaType.valueOf(rdfFormat.getHeaderString()));
```

Comment:
Can't you use the 'contentType' parameter that you receive here? Now you convert this param to a lang and then extract it from the lang and map it back to your MediaType that you started with..